db_sql.go

```diff
@@ -42,12 +42,16 @@ const (
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 type sqlUserStoreDB struct {
-	db             *sql.DB
-	passwordExpiry time.Duration
+	db                   *sql.DB
+	passwordExpiry       time.Duration
+	enableAccountLocking bool
```

Comment:
Does enableAccountLocking and maxFailedPasswords need to be here? I think we don't need this state inside the UserStore. I think it can just live at the higher levels (eg inside authaus.Central)